src/cascadia/UnitTests_TerminalCore/TerminalBufferTests.cpp

```diff
@@ -233,3 +246,349 @@ void TerminalBufferTests::DontSnapToOutputTest()
     VERIFY_ARE_EQUAL(TerminalViewHeight, seventhView.BottomExclusive());
     VERIFY_ARE_EQUAL(TerminalHistoryLength, term->_scrollOffset);
 }
+
+void TerminalBufferTests::_SetTabStops(std::list<short> columns, bool replace)

```

Comment:
I honestly didn't expect the whole chunk of tests just pass without too much modification. I'm glad they do 😄